src/test/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/integration/listener/CourtHearingEventListenerTest.kt

```diff
@@ -139,6 +148,40 @@ class CourtHearingEventListenerTest : IntegrationTestBase() {
     assertThat(savedResult.success).isTrue()
   }
 
+  @Test
+  fun `Will log event for malformed payload`() {
+    courtHearingEventListener.onDomainEvent("BAD JSON")
```

Comment:
Testing CourtHearingEventListener directly, as getting wired test failure when testing via SQS and telemetry client verify are called but testing randomly thinks it is not called